### PR TITLE
add shape description in accuracy api doc 

### PR DESCRIPTION
**Pull request by slf12**
add shape description in accuracy api doc

### CODE REVIEW

**Comment by luotao1** [comment]
PR描述里不要贴内网链接，请将对应的内容复制过来即可。

**Comment by slf12** [comment]
> PR描述里不要贴内网链接，请将对应的内容复制过来即可。

done

File: python/paddle/fluid/layers/metric_op.py [hunk 1]
```diff
@@ -39,7 +39,8 @@ def accuracy(input, label, k=1, correct=None, total=None):
 
     Args:
         input(Variable): The input of accuracy layer, which is the predictions of network. A LoDTensor or Tensor with type float32,float64.
-        label(Variable): The label of dataset.  LoDTensor or Tensor with type int32,int64.
+            Shape is ``[sample_number, class_dim]`` .
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/metric_op.py]: Shape->The shape

File: python/paddle/fluid/layers/metric_op.py [hunk 2]
```diff
@@ -39,7 +39,8 @@ def accuracy(input, label, k=1, correct=None, total=None):
...
+        label(Variable): The label of dataset.  LoDTensor or Tensor with type int32,int64. Shape is ``[sample_number, 1]`` .
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/metric_op.py]: Shape->The shape

File: python/paddle/fluid/layers/metric_op.py (hunk 1 above)
Comment (slf12) [reply python/paddle/fluid/layers/metric_op.py]: done

File: python/paddle/fluid/layers/metric_op.py (hunk 2 above)
Comment (slf12) [reply python/paddle/fluid/layers/metric_op.py]: done

**Review by Boyan-Liu — APPROVED** [review]
LGTM

**Review by baiyfbupt — APPROVED** [review]
LGTM

**Pull request merged**